crypto/cmp/cmp_msg.c

```diff
@@ -454,9 +454,9 @@ OSSL_CMP_MSG *ossl_cmp_certreq_new(OSSL_CMP_CTX *ctx, int type,
 }
 
 OSSL_CMP_MSG *ossl_cmp_certrep_new(OSSL_CMP_CTX *ctx, int bodytype,
-                                   int certReqId, OSSL_CMP_PKISI *si,
-                                   X509 *cert, STACK_OF(X509) *chain,
-                                   STACK_OF(X509) *caPubs, int encrypted,
+                                   int certReqId, const OSSL_CMP_PKISI *si,
+                                   X509 *cert, const X509 *encryption_recipient,
```

Comment:
Why are you adding a `encryption_recipient` parameter here?

Reply:
I'm replacing the `encrypted` parameter by something that will be much more useful.
Yet note that, as documented, the encryption of newly enrolled certificates it is not yet supported;
if the parameter is given anyway this is currently reported as error.